### ISSUE TITLE
We should add local `name:xx` tags

### ISSUE THREAD

**Issue by bhousel** [osmlab/name-suggestion-index#2245]
In working with KeepRight data a lot lately, I notice that it sometimes flags a warning like this:

> It would be nice if this node had an additional tag 'name:XX=Chick-fil-A' where XX shows the language of its name 'Chick-fil-A'.

<img width="300" alt="screenshot 2019-01-05 01 30 33" src="https://user-images.githubusercontent.com/38784/50721201-820d8980-1089-11e9-92f9-fcad5e872c97.png">

It seems like the warning is caused if someone adds a `name:xx` tag and a `name` and they don't match. 

<img width="300" alt="screenshot 2019-01-05 01 32 22" src="https://user-images.githubusercontent.com/38784/50721218-d57fd780-1089-11e9-885c-49e3bb3ed1f4.png">
<img width="300" alt="screenshot 2019-01-05 01 32 27" src="https://user-images.githubusercontent.com/38784/50721220-d9abf500-1089-11e9-8c0c-4d534d64d115.png">

It is only a warning , but  I can understand why it would be nice in this case to add a `name:en` tag, so that people know what language `name` is in or where it came from.

We have a lot of names like this in the index.  Our Contributing guidelines even encourage it:

```js
  "amenity/fast_food|マクドナルド": {         // Identifier like "key/value|name"
    "count": 1445,
    "countryCodes": ["jp"],                 // Optional `countryCodes` - array of countries where this entry is used
    "tags": {
      "amenity": "fast_food",
      "brand": "マクドナルド",                // `brand` - Brand name in the local language (Japanese)
      "brand:en": "McDonald's",             // `brand:en` - For non-English brands, tag the English version too
      "brand:wikidata": "Q38076",           // `brand:wikidata` - Same Universal wikidata identifier
      "brand:wikipedia": "ja:マクドナルド",   // `brand:wikipedia` - Reference to Japanese Wikipedia
      "cuisine": "burger",
      "name": "マクドナルド",                 // `name` - Display name, also in the local language (Japanese)
      "name:en": "McDonald's"               // `name:en` - For non-English names, tag the English version too
     // SHOULD WE ADD "name:ja" TOO??
    }
  },
```

**Comment by SilentSpike** [comment]
I can't think of any strong argument against it 👍

**Comment by matkoniecz** [comment]
from what I see there is no pending issue to remove this complaint in https://github.com/keepright/keepright/issues (I checked as Keep Right is no longer developed).

**Comment by matkoniecz** [comment]
JOSM also complains (at information level)

For example `shop=car`, `name=ホンダ`, `name:en=Honda` node gets

> Missing name:* translation - Missing name:*=ホンダ. Add tag with correct language key. (1)

So I think that it would be OK to add also `name:<local>`.

**Comment by sommerluk** [comment]
For “Burger King” for example it's the same name in French, German and Englisch. In Germany I would expect rather to find a :de while in France I would expect rather a :fr on an object…

**Comment by bhousel** [comment]
> For “Burger King” for example it's the same name in French, German and Englisch. In Germany I would expect rather to find a :de while in France I would expect rather a :fr on an object…

For that example, we wouldn't set _any_ `name:xx` tag.

This issue is saying _if_ we set a `name:xx` tag, make sure there exists a `name:xx` tag that matches the `name` tag.

**Comment by SilentSpike** [comment]
Oh that makes more sense, I also misunderstood this and thought those warnings wanted `name:xx` on everything 😆

**Comment by bhousel** [comment]
I've updated the CONTRIBUTING doc and also added `name:xx` and `brand:xx` tags for situations where the language used is unambiguous. Rule going forward is as stated above: 

* "If we set a `name:xx` tag, make sure there exists a `name:xx` tag that matches the `name` tag."
* "If we set a `brand:xx` tag, make sure there exists a `brand:xx` tag that matches the `brand` tag."
* etc..

This should keep people happy!   It means there is a lot of work to do in the index to add these tags, but I could add a build warning to assist cleanup.
